docker/checkout.bash

```diff
@@ -45,16 +45,16 @@ cd ${dir}/src
 if [[ "$BRANCH" == "develop" ]]; then
       cd /home/carma/
       git clone https://github.com/usdot-fhwa-stol/autoware.ai.git --depth 1 --branch carma-"$BRANCH"
-      cd ${dir}/src 
+      cd /home/carma/.base-image/workspace/src
```

Comment:
Is this the only change. If so, I think the calling script just needs to specify this directory. We probably also have to fix the else statement to clone into this directory too. When I say "this directory" I mean the one provided as an argument to the script `-r <dir>` and set to the local variable ${dir}

Reply:
Addressed in [472a285](https://github.com/usdot-fhwa-stol/carma-msgs/pull/234/commits/472a28514a86bc099e1c8a88374a0c4f8c1fc486)